src/Components/Books.js

```diff
@@ -12,19 +13,43 @@ const Books = (props) => {
   };
 
   return (
-    <div className="d-flex flex-column align-items-center">
-      <div className="border border-secondary rounded  w-75 my-3 ps-5">
-        <p className="text-left">{type}</p>
-        <p>{title}</p>
-        <p>{author}</p>
-
-        <button
-          type="button"
-          className="btn btn-primary"
-          onClick={clickHandler}
-        >
-          Remove
-        </button>
+    <div>
+      <div className="book-card">
+        <div>
+          <p className="category">{type}</p>
+          <p className="Title">{title}</p>
+          <p className="author">{author}</p>
+          <button
+            type="button"
+            className="btn btn-primary"
+            onClick={clickHandler}
+          >
+            Remove
+          </button>
```

Comment:
- [ ] Please ensure the book card design matches the Zeplin design:

- Your design
<img width="881" alt="image" src="https://user-images.githubusercontent.com/77190203/194715672-70fdeb70-ee34-4c85-ad7e-2630eb2ab2bb.png">

- Zeplin design
<img width="506" alt="image" src="https://user-images.githubusercontent.com/77190203/194715490-eb5fa399-e7ba-4639-949f-2febdb3b257b.png">